### PR TITLE
Tweak wording of summary for some settings

### PR DESCRIPTION
**Pull request by dbr**
Minor improvement to the grammar in some of the settings text

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -543,17 +543,17 @@ Please report any problems you encounter via Github.</string>
 
     <string name="use_root_title">Run Syncthing as Superuser</string>
 
-    <string name="use_root_summary">Running Syncthing as root allows it to write to folders Android normally restricts to be readonly accessed. Use this feature cautious.</string>
+    <string name="use_root_summary">Running Syncthing as root allows it to write to folders Android normally restricts to be readonly accessed. Use this feature with caution.</string>
 
     <string name="launcher_show_camera_icon_title">Show camera icon in launcher</string>
 
-    <string name="launcher_show_camera_icon_summary">Disable this option if you do not like to use the \'Syncthing Camera\' feature. This does not work with launcher apps that do not support hiding icons.</string>
+    <string name="launcher_show_camera_icon_summary">Disable this option if you do not like to use the \'Syncthing Camera\' feature. This only works launcher apps that support hiding icons.</string>
```
Comment (Catfriend1) [inline app/src/main/res/values/strings.xml:550]: Hi,

thanks for your feedback. I would suggest : "... This only works with launcher apps ...".
Pretty good. Well done! :-)

Kind regards,
Catfriend1

P.S. Other languages need to be translated on Transifex. Do you like to also participate there, then please let me know your TF nickname so I can accept the request.

Comment (dbr) [reply app/src/main/res/values/strings.xml]: Oops - fixed typo!

Don't think I can be of much help translating as I know only English!

**Review by Catfriend1 — APPROVED** [review]
lgtm?

**Comment by Catfriend1** [comment]
Thanks for contributing :-)

**Pull request merged**